test/integration_windows_test.go

```diff
@@ -233,7 +233,11 @@ func CopyEnvironmentToPath(t *testing.T, environmentPath, targetPath string) {
 }
 
 func CopyEnvironmentWithTflint(t *testing.T, environmentPath string) string {
-	tmpDir, err := os.MkdirTemp("", "terragrunt-test")
+	currentDir, err := os.Getwd()
+	if err != nil {
+		t.Fatalf("Failed to get current directory: %v", err)
+	}
+	tmpDir, err := os.MkdirTemp(currentDir, "terragrunt-test")
```

Comment:
NIT: Why aren't we using `t.Tempdir` here?

Reply:
t.Tempdir create temporary directory in C: ... but windows tflint tests are executed in D:, which leads to issues on tflint parent detection